doc/source/development.rst

```diff
@@ -99,6 +99,47 @@ directory will take effect without reinstalling the package.
 
 .. warning:: if you run ``python setup.py install``, files will be copied from the Ray directory to a directory of Python packages (``/lib/python3.6/site-packages/ray``). This means that changes you make to files in the Ray directory will not have any effect.
 
+Building Ray on Windows (full)
+------------------------------
+
+**Requirements**
```

Comment:
Can you add links here where people can download these?

Reply:
Also we should point out that bazel 3.2.0 is required.

Reply:
Added bazel to the requirements list.

Reply:
Links added.